### ISSUE TITLE
R79 Hardening and monitoring of exposed services

### ISSUE THREAD

**Issue by obp-anssi** [yuumasato/scap-security-guide#17]
The existing rules are not enough to ensure that the exposed services are properly hardened and monitored as the recommendation states. SELinux only, without ensuring that the proper profiles and labels are set, do not ensure all the points of the recommendation.

In my opinion, detecting the exposed services also seems difficult to do on an automated fashion, so I suggest changing this one to "manual". Feel free to contradict me and to bring new rules.

**Comment by yuumasato** [comment]
I agree with your assessment here, the recommendation's status is `automated` because the rule selection goes up to what is reasonably doable.
Maybe this recommendation should be switched to `partial`.
